subsys/espi/ndsx_espi.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/espi.h>
+
+#include "x86_non_dsx.h"
+
+static const struct device *espi_dev;
+static struct espi_callback espi_bus_cb;
+static struct espi_callback espi_chan_cb;
+static struct espi_callback espi_vw_cb;
+
+static void espi_bus_reset(void)
```

Comment:
The code in this function sees like it it target to the power sequencing part. I think this code should go in the power sequence folder (maybe as a separate file that has to do with eSPI though)

Reply:
ACK